cmd/cli/get.go

```diff
@@ -80,4 +85,6 @@ func init() {
 	rootCmd.AddCommand(getCmd)
 	getCmd.Flags().StringVar(&getArgs.KubeconfigPath, "kubeconfig", "", "Path to kubeconfig")
 	getCmd.Flags().StringVar(&getArgs.Format, "format", "table", "The format in which to display results (currently only 'table' supported)")
+	getCmd.Flags().StringVar(&upsertArgs.AsUser, "as", "", "Username to impersonate for the operation")
+	getCmd.Flags().StringSliceVar(&upsertArgs.AsGroups, "as-group", []string{}, "Group to impersonate for the operation, this flag can be repeated to specify multiple groups")
```

Comment:
Will a comma separated list work in this case as well? if thats the case description can just say "List of groups to impersonate"

Reply:
@eytan-avisror actually not sure off hand how to write a test for this, there isnt anything in place in the tests where the cli is being called. this works though
```
 go run main.go upsert --as me --as-group "system:masters" --as-group "sudoers"  --maproles --rolearn arn:aws:iam::xxxxxxxxxxxxxxxx:role/test_team   --append --groups testing --username foo --update-username=false
role arn:aws:iam::xxxxxxxxxxxxxxxx:role/test_team has been updated
```